docs/pipelineruns.md

```diff
@@ -9,28 +9,28 @@ weight: 500
 <!-- toc -->
 - [PipelineRuns](#pipelineruns)
```

Comment:
not a big deal, but generally it's preferred for a PR to do only one thing, so these formatting changes might make more sense as part of a separate PR.

Reply:
thanks this logic. raise https://github.com/tektoncd/pipeline/issues/4842 for a separate task.